app/views/admin/statistics_announcements/_release_date.html.erb

```diff
@@ -0,0 +1,57 @@
+<%= render "components/datetime_fields", {
+  date_heading: "Release date (required)",
+  field_name: "release_date",
+  prefix: "statistics_announcement[current_release_date_attributes]",
+  id: "statistics_announcement_current_release_date_release_date",
+  year: {
+    start_year: Date.today.year,
+    end_year: Date.today.year + 5,
+    id: "release_date(1i)",
+    value: statistics_announcement.release_date&.year
+  },
+  month: {
+    id: "release_date(2i)",
+    value: statistics_announcement.release_date&.month
+  },
+  day: {
+    id: "release_date(3i)",
+    value: statistics_announcement.release_date&.day
+  },
+  hour: {
+    id: "release_date(4i)",
+    value: statistics_announcement.release_date&.hour
+  },
+  minute: {
+    id: "release_date(5i)",
+    value: statistics_announcement.release_date&.min
+  },
+  error_items: errors_for(statistics_announcement.errors, :release_date),
+}
+%>
+
+<%= render "govuk_publishing_components/components/radio", {
+  heading: "Delivery date",
+  name: "statistics_announcement[current_release_date_attributes][precision]",
+  id: "statistics_announcement_current_release_date_precision",
+  items: [
+    {
+      value: 0,
+      text: "Exact date (provisional)"
+    },
+    {
+      value: 1,
+      text: "Within one month (provisional)"
+    },
+    {
+      value: 2,
+      text: "Within two months (provisional)",
+      checked: true
```

Comment:
I think this needs to be 
```
 checked: statistics_announcement.current_release_date.precision == 2
```

and the same for each of the radios. 

so 

```
 checked: statistics_announcement.current_release_date.precision == 0
 checked: statistics_announcement.current_release_date.precision == 1
 checked: statistics_announcement.current_release_date.precision == 3
```

etc.

Reply:
Yes, good catch. Not sure if this behaviour is vital for the new page, but is definitely necessary for the update release date page.

Reply:
Implemented as suggested ✅